components/Article/Article.tsx

```diff
@@ -33,10 +33,12 @@ export const Article: FC<IArticleProps> = ({ article, author, redirectUrl, ...pr
   useEffect(() => {
     const contentContainer = contentRef.current;
     const contentMaxChar = 500;
+
     if (contentContainer) {
       const span = document.createElement('span');
-      span.innerHTML =
-        content.length > contentMaxChar ? `${content.slice(0, contentMaxChar)}...` : content;
+      span.innerHTML = content;
```

Comment:
please use css styles like
    display: -webkit-box;
    -webkit-line-clamp: 2;
    -webkit-box-orient: vertical;
    overflow: hidden;

Reply:
Itʼs already been there